app/src/main/java/edu/uc/haflga/financialpal/ui/income/IncomeFragment.kt

```diff
@@ -29,4 +25,8 @@ class IncomeFragment : Fragment() {
         // TODO: Use the ViewModel
     }
 
+    companion object {
+        fun newInstance() = IncomeFragment()
+    }
+
```

Comment:
Moved companion object to bottom: https://kotlinlang.org/docs/reference/coding-conventions.html#source-code-organization